### DISCUSSION TITLE
mobile - Quill has dependency version conflict

### DISCUSSION THREAD

**Discussion by klivin** [singerdmx/flutter-quill#837] [Mobile]
Receive error when I try to update a quill dependency, photo_view to version 0.14.0:

Because flutter_quill 5.0.2 depends on photo_view ^0.13.0 and no versions of flutter_quill match >5.0.2 <6.0.0, flutter_quill ^5.0.2 requires photo_view ^0.13.0.

Workaround is to use an outdated photo_view explicitly stating in yaml:
  photo_view: ^0.13.0

**Comment by mark8044** [comment]
Fixed here: https://github.com/singerdmx/flutter-quill/pull/839
